src/pattern-library/components/PlaygroundApp.tsx

```diff
@@ -19,130 +91,125 @@ export default function PlaygroundApp({
-                Component Library
-              </Link>
-            </h1>
-          </div>
-          <nav id="nav" className="pt-8 pb-16 space-y-4 mr-4">
-            <NavHeader>Foundations</NavHeader>
-            <NavList routes={getRoutes('foundations')} />
+    <Router base={baseURL}>
+      <main className="max-w-[1200px] mx-auto">
+        <div className="md:grid md:grid-cols-[2fr_5fr]">
+          <div className="md:h-screen md:sticky md:top-0 overflow-scroll">
+            <div className="md:sticky md:top-0 h-16 px-6 flex items-center bg-slate-0 border-b">
+              <h1 className="text-lg">
+                <Link href={baseURL + '/'} classes="grow flex gap-x-2">
+                  <LogoIcon />
+                  Component Library
+                </Link>
+              </h1>
+            </div>
+            <nav id="nav" className="pt-8 pb-16 space-y-4 mr-4">
+              <NavHeader>Foundations</NavHeader>
+              <NavList>
+                {getRoutes('foundations').map(route => (
+                  <NavLink key={route.title} route={route} />
+                ))}
+              </NavList>
 
-            <NavHeader>Components</NavHeader>
+              <NavHeader>Components</NavHeader>
 
-            {groupKeys.map(group => {
-              return (
-                <NavSection
-                  key={group}
-                  title={componentGroups[group] as string}
-                >
-                  <NavList routes={getRoutes(group)} />
-                </NavSection>
-              );
-            })}
+              {groupKeys.map(group => {
+                return (
+                  <NavSection
+                    key={group}
+                    title={componentGroups[group] as string}
+                  >
+                    <NavList>
+                      {getRoutes(group).map(route => (
+                        <NavLink key={route.title} route={route} />
+                      ))}
+                    </NavList>
+                  </NavSection>
+                );
+              })}
 
-            {extraRoutes.length > 0 && (
-              <>
-                <NavHeader>{extraRoutesTitle}</NavHeader>
-                <NavList routes={customRoutes} />
-              </>
-            )}
-          </nav>
+              {extraRoutes.length > 0 && (
+                <>
+                  <NavHeader>{extraRoutesTitle}</NavHeader>
+                  <NavList>
+                    {customRoutes.map(route => (
+                      <NavLink key={route.title} route={route} />
+                    ))}
+                  </NavList>
+                </>
+              )}
+            </nav>
+          </div>
+          <div className="bg-white pb-16">
+            <Switch>
+              {pageRoutes}
+              <Route>
+                <Library.Page title=":( Sorry">
+                  <h1 className="text-2xl">Page not found</h1>
+                </Library.Page>
+              </Route>
+            </Switch>
```

Comment:
`<Switch>` is a `wouter` component that will render (only) the first matching `<Route>` inside of it.